### PR TITLE
Address GUI changes that do not require access to improved Schema

### PR DESCRIPTION
**Pull request by jchate6**
See https://drive.google.com/file/d/19vcnbDyestDX6gfIY0rxpe4-sYpfLx30/view

### CODE REVIEW

File: src/router/index.ts [hunk 1]
```diff
@@ -45,7 +46,10 @@ const routes = [
     path: "/report-availability",
     name: "report-availability",
     component: SubmitTelescopeEvents},
-
+  {
+    path: "/about-hermes",
```
Comment (jnation3406) [inline src/router/index.ts:50]: i think it should just be `/about`

File: src/views/AboutHermes.vue [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+                    </b-collapse>
+                </b-card>
+                <b-card no-body class="mb-1" border-variant="primary">
+                    <b-card-header header-tag="header" class="p-1" role="tab">
+                        <b-button block v-b-toggle.accordion-2 variant="primary">How do I log in?</b-button>
+                    </b-card-header>
+                    <b-collapse id="accordion-2" visible accordion="my-accordion" role="tabpanel">
+                        <b-card-body>
+                            <b-card-text>
+                                To log into HERMES, you have to do a bunch of SCiMMA stuff. Just email Lindy, and he'll help you figure it out. :D
```
Comment (jnation3406) [inline src/views/AboutHermes.vue:24]: placeholder for andy?

File: src/views/ViewMessages.vue [hunk 3]
```diff
@@ -82,6 +82,21 @@
         </div>
       </template>
     </b-table>
+    <b-row>
+      <b-col class="col-md-8 ml-auto">
+        <!-- Pagination -->
+        <ocs-pagination
```
Comment (jnation3406) [inline src/views/ViewMessages.vue:88]: I'm kinda confused what this is doing here vs the ocs-pagination on line 21? Is it just to have the pagination controls on the top and bottom or something?

File: src/views/AboutHermes.vue (hunk 2 above)
Comment (jchate6) [reply src/views/AboutHermes.vue:24]: Yeah, I don't actually know what should go on the about page?
Andy says "Version number, where to email, and documentation" but we never actually populated the version number and the email is in the footer, so...Figured this needed more thought.

File: src/views/ViewMessages.vue (hunk 3 above)
Comment (jchate6) [reply src/views/ViewMessages.vue:88]: He wants to move pagination to the bottom and replace the top one with a search bar. 
I left both for the moment.

File: src/views/AboutHermes.vue (hunk 2 above)
Comment (phycodurus) [reply src/views/AboutHermes.vue:24]: lol :-)

**Review by phycodurus — APPROVED** [review]

**Review by jnation3406 — APPROVED** [review]

**Pull request merged**